pmdarima/arima/_auto_solvers.py

```diff
@@ -56,6 +57,7 @@ def __init__(self, y, xreg, start_params, trend, method, transparams,
         self.scoring_args = scoring_args
         self.information_criterion = information_criterion
         self.with_intercept = with_intercept
+        self.kwargs = dict() if not kwargs else kwargs
```

Comment:
Is there a reason to use `dict()` vs `{}`? I know `{}` is faster (https://stackoverflow.com/a/5790954)

Reply:
`{}` can be a [set literal](https://stackoverflow.com/questions/36674083/why-is-it-possible-to-replace-sometimes-set-with) 😬

Reply:
The way I understand it, Python will _always_ interpret `{}` as an empty dictionary. So you end up with the following:

```python
example1 = {}  # Empty dict
example2 = {1: 'test'}  # Dict with one item
example3 = set()  # Empty set
example4 = {1, 2, 3}  # Set with 3 items
```
[The official docs even define `{}` as an empty dict.](https://docs.python.org/3.7/library/stdtypes.html#truth-value-testing) The _only_ definition of an empty set is `set()`

Am I off base? I don't think the fraction of a second is a huge deal, but we _do_ get a lot of questions about performance, so I figured I would bring it up!